docs/cspell.json

```diff
@@ -838,6 +842,72 @@
     "snowsql",
     "spacectl",
     "spacelift",
+    "specallow",
+    "specallowdb",
+    "specallowimpersonate",
+    "specallowjoin",
+    "specallowkubernetes",
+    "specallowrequest",
+    "specallowrequestclaims",
+    "specallowrequestthresholds",
+    "specallowrequire",
+    "specallowreview",
+    "specallowrules",
+    "specallowspiffe",
+    "specassertion",
+    "specattributes",
+    "specaudit",
+    "specauditnotifications",
+    "specauditrecurrence",
+    "specazure",
+    "specazureallow",
+    "speccircleci",
+    "speccircleciallow",
+    "specclaims",
+    "specclient",
+    "speccloud",
+    "specdeny",
+    "specdenydb",
+    "specdenyimpersonate",
+    "specdenyjoin",
+    "specdenykubernetes",
+    "specdenyrequest",
+    "specdenyrequestclaims",
+    "specdenyrequestthresholds",
+    "specdenyrequire",
+    "specdenyreview",
+    "specdenyrules",
+    "specdenyspiffe",
+    "specgcp",
+    "specgcpallow",
+    "specgithub",
+    "specgithuballow",
+    "specgitlab",
+    "specgitlaballow",
+    "specgrants",
+    "speckubernetes",
+    "speckubernetesallow",
+    "speckubernetesstatic",
+    "specmappings",
+    "specmembership",
+    "specoidc",
+    "specoptions",
+    "specoptionscert",
+    "specoptionsidp",
+    "specoptionsidpsaml",
+    "specoptionsrecord",
+    "specowner",
+    "specowners",
+    "specownership",
+    "specrotation",
+    "specrotationschedule",
+    "specsaml",
+    "specsigning",
+    "specspacelift",
+    "specspaceliftallow",
+    "specteams",
+    "spectpm",
+    "spectpmallow",
```

Comment:
This won't scale, can we skip the spell check on the links or those pages instead?

Reply:
I have a PR to address this on `master` here: https://github.com/gravitational/teleport/pull/46259